x-pack/test/api_integration/apis/telemetry/fixtures/basiccluster.json

```diff
@@ -176,6 +174,15 @@
       "apm": {
         "found": true
       },
+      "data": [
+        {
+          "dataset": { "name": "apm", "type": "unknown" },
+          "shipper": "unknown",
+          "doc_count": 1,
+          "index_count": 1,
+          "size_in_bytes": 5136
+        }
+      ],
```

Comment:
For monitoring, we know we can't get the `dataset.type` nor the `shipper` because we don't have access to the `mappings`:
- Should we not report those fields instead of reporting `"unknown"`?
- Should we add `dataType` and `shipper` to the list of patterns when known (similar to the pattern `*logs*`)?

Reply:
Maybe we will be able to provide this info in monitoring by tackling https://github.com/elastic/kibana/issues/68998